### PR TITLE
structural changes

### PR DESCRIPTION
**Pull request by Satendra-SR**
### Changes include:

- Updated `composer.json` file to install WordPress and `wordfence` and `WP Smush ` plugin.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,27 +1,38 @@
 {
+    "name": "ColoredCow/wordpress-init",
+    "description": "Boilerplate for Wordpress",
```
Comment (abhishek-pokhriyal) [inline composer.json]:
```suggestion
    "description": "Boilerplate for WordPress",
```

File: public/wp-content/themes/coloredcow/package.json [hunk 2]
```diff
@@ -1,6 +1,12 @@
 {
   "name": "ColoredCow",
   "version": "1.0.0",
```
Comment (mohitgusain) [inline public/wp-content/themes/coloredcow/package.json]: can we also change its version as we are upgrading it? Discuss it with others once.

File: composer.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Gauravbhatt19) [reply composer.json]: Done with the changes, merging the Pull request...

File: public/wp-content/themes/coloredcow/package.json [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
+  "description": "ColoredCow package for wordpress installation",
```
[the remark this answers is not in the document]
Comment (Gauravbhatt19) [reply public/wp-content/themes/coloredcow/package.json]: Done with the changes, merging the Pull request...

File: public/wp-content/themes/coloredcow/package.json (hunk 2 above)
Comment (Gauravbhatt19) [reply public/wp-content/themes/coloredcow/package.json]: All agreed on upgrading the version to `1.0.1`.

File: composer.json (hunk 1 above)
Comment (Gauravbhatt19) [reply composer.json]: Done with the changes!

**Pull request merged**